bot.py

```diff
@@ -32,6 +44,20 @@
             await client.replace_roles(author, *new_roles)
             await client.send_message(message.channel, "Changed your color to " + words[1])
 
+async def change_song(message):
+    global current_track
+    global elevator
+    try:
+        song = message.content.split(' ')[1]
+    except IndexError:
+        print('The current song is : {}'.format(get_video_title(current_track)))
```

Comment:
This won't send to the discord channel, it'll print to the console. Is that intentional?